inc/widgets/post-loop.php

```diff
@@ -180,18 +180,18 @@ function widget( $args, $instance ) {
 			echo $args['before_title'] . $instance['title'] . $args['after_title'];
 		}
 		
-		global $more; $old_more = $more; $more = empty($instance['more']);
+		global $more; $old_more = $more; $more = empty( $instance['more'] );
 		self::$rendering_loop = true;
 		self::$current_loop_instance = $instance;
 		self::$current_loop_template = $instance['template'];
-		if(strpos('/'.$instance['template'], '/content') !== false) {
+		if ( strpos( '/'.$instance['template'], '/content' ) !== false) {
 			while( have_posts() ) {
 				the_post();
 				locate_template($instance['template'], true, false);
 			}
-		}
-		else {
-			locate_template($instance['template'], true, false);
+		} elseif ( file_exists( $instance['template'] ) ) {
+				load_template( $instance['template'], false );
+			}
```

Comment:
I assume there isn't a syntax error happening here, but this indenting doesn't make sense. Looks like this closing `}` shouldn't be here. I can't quite figure out what's going on.

Reply:
Ah yeah, it will cause a syntax error. I missed this before merging the PR, then fixed it in the release branch locally, but didn't push the commit.